composeApp/src/androidMain/kotlin/id/andriawan/cofinance/auth/GoogleAuthManager.android.kt

```
@@ -0,0 +1,78 @@
+
+            try {
+                val result = credentialManager.getCredential(
+                    context = context,
+                    request = request
+                )
+                handleSignInResult(result)
+            } catch (e: GetCredentialCancellationException) {
+                GoogleAuthResult.Cancelled
+            } catch (e: GetCredentialException) {
+                GoogleAuthResult.Error(e.message ?: "Failed to get credentials", e)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)